jsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "checkJs": true,  /* Typecheck .js files. */
```

Comment:
Some more comments here to remove. 

Maybe do Another Check through the file for more. 
Thanks @steaward

Reply:
These were actually auto-generated by the Yo Code extension generator used by the tutorial. If we want to follow @humphd 's suggestion of keeping json files free of comments, this can be removed...